app/controllers/api/v1/items_controller.rb

```diff
@@ -8,6 +8,20 @@ def create
     end
   end
 
+  def update
+    begin
+      # return render status: 400 unless item_params.keys.all? { |k| Item.column_names.include?(k) }
+      item = Item.find(params[:id])
+      item.update_attributes!(item_params)
```

Comment:
Why `!` on this method? I'd follow the `if item.save` pattern instead.

Reply:
I'm going to save this for a refactor later.